### PR TITLE
docs: Update rune 0.2.0 release docs.

### PR DESCRIPTION
**Pull request by hustliyilin**
1. Add docs/running_rune_with_occlum_bundle.md.
2. Remove docs/running_rune_with_docker.md.
3. Update skeleton README.md to add way to run rune with skeleton bundle.
4. Update README.md of inclavare-containers.
5. Add docs/running_rune_with_occlum.md.

Signed-off-by: Yilin Li <YiLin.Li@linux.alibaba.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,12 +49,14 @@ In addition, you can write your own enclave runtime with any programming languag
...
-
 ### Run skeleton
 Skeleton is an example of enclave runtime, interfacing with Enclave Runtime PAL API for easy interfacing with `rune`.  Skeleton sample code is helpful to write your own enclave runtime.
 
 Please refer to [this guide](https://github.com/alibaba/inclavare-containers/blob/master/rune/libenclave/internal/runtime/pal/skeleton/README.md) to run skeleton with `rune`.
 
+### Run OCI bundle
+Please refer to [this guide](https://github.com/alibaba/inclavare-containers/blob/master/docs/running_rune_with_skeleton_bundle.md) to run `skeleton bundle` with `rune`.
+
+In addition, you can refer to [this guide](https://github.com/alibaba/inclavare-containers/blob/master/docs/running_rune_with_skeleton_bundle.md) to run more useful Occlum library OS.
```
Comment (jiazhang0) [inline README.md:60]: This link still points to skeleton bundle usage rather than occulm.

File: docs/running_rune_with_skeleton_bundle.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Quick Start: Running rune with skeleton bundle
```
Comment (jiazhang0) [inline docs/running_rune_with_skeleton_bundle.md:1]: Merge this doc into skeleton README plz.

File: rune/libenclave/internal/runtime/pal/skeleton/README.md [hunk 3]
```diff
@@ -64,7 +61,66 @@ Runtimes: rune runc
 ```shell
 docker run -it --rm --runtime=rune \
   -e ENCLAVE_TYPE=intelSgx \
-  -e ENCLAVE_RUNTIME_PATH=/run/rune/liberpal-skeleton.so \
+  -e ENCLAVE_RUNTIME_PATH=/usr/lib/liberpal-skeleton.so \
```
Comment (jiazhang0) [inline rune/libenclave/internal/runtime/pal/skeleton/README.md:64]: Should be /usr/lib64/liberpal-skeleton.so

Comment (hustliyilin) [reply rune/libenclave/internal/runtime/pal/skeleton/README.md:64]: For Ubuntu, it doesn't have `/usr/lib64`, but has '/usr/lib'

Comment (hustliyilin) [reply rune/libenclave/internal/runtime/pal/skeleton/README.md:64]: > Should be /usr/lib64/liberpal-skeleton.so

For Ubuntu, it doesn't have `/usr/lib64`, but has `/usr/lib`

Comment (jiazhang0) [reply rune/libenclave/internal/runtime/pal/skeleton/README.md:64]: > > Should be /usr/lib64/liberpal-skeleton.so
> 
> For Ubuntu, it doesn't have `/usr/lib64`, but has `/usr/lib`

I mean you documented /usr/lib64 in previous section:
```shell
 cd "${path_to_inclavare_containers}/rune/libenclave/internal/runtime/pal/skeleton"
 make
 cp liberpal-skeletion.so /usr/lib64
 ```

So you need to keep consistence, and should document well for both ubuntu and centos.

Comment (hustliyilin) [reply rune/libenclave/internal/runtime/pal/skeleton/README.md:64]: OK

Comment (hustliyilin) [reply rune/libenclave/internal/runtime/pal/skeleton/README.md:64]: > > > Should be /usr/lib64/liberpal-skeleton.so
> > 
> > 
> > For Ubuntu, it doesn't have `/usr/lib64`, but has `/usr/lib`
> 
> I mean you documented /usr/lib64 in previous section:
> 
> ```shell
>  cd "${path_to_inclavare_containers}/rune/libenclave/internal/runtime/pal/skeleton"
>  make
>  cp liberpal-skeletion.so /usr/lib64
> ```
> 
> So you need to keep consistence, and should document well for both ubuntu and centos.

OK

**Pull request merged**